specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2020-11-01-preview/containerregistry.json

```diff
@@ -4341,6 +4345,25 @@
         }
       }
     },
+    "ExportPolicy": {
+      "description": "The export policy for a container registry.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The value that indicates whether the policy is enabled or not.",
```

Comment:
If this description is specific to export policy, will adding the work export be useful like 
"The value that indicates whether the export policy is enabled or not."

Reply:
@mnltejaswini I was keeping in line with other policy status descriptions. See here: https://github.com/Azure/azure-rest-api-specs/pull/14559/files#diff-f7ab10e18ad7fede7b20d3325bbeed68c1332b3eb0d376ce0c779382fe054520R4334